routing/routes.go

```diff
@@ -793,6 +793,27 @@ func NewRoutes() (r *Routes) {
 			handlers.NamespaceHealth,
 			true,
 		},
+		// swagger:route GET /namespaces/{namespace}/validations namespaces namespaceValidations
+		// ---
+		// Get validation summary for all objects in the given namespace
+		//
+		//     Produces:
+		//     - application/json
+		//
+		//     Schemes: http, https
+		//
+		// responses:
+		//      200: namespaceValidationSummaryResponse
+		//      400: badRequestError
+		//      500: internalError
+		//
+		{
+			"NamespaceValidationSummary",
+			"GET",
+			"/api/namespaces/{namespace}/validations",
```

Comment:
@lucasponce Perhaps this should be /validationsummary ?

Reply:
As there is no other "/validations" endpoint, this is fine.
For consistency we have used one single term to refer the endpoints (unless there  is a conflict).

Reply:
OK, just making sure because the other comments were to change from validations to validationSummaries.